src/reanimated2/layoutReanimation/web/componentUtils.ts

```diff
@@ -290,4 +290,10 @@ export function handleExitingAnimation(
     // Given that this function overrides onAnimationEnd, it won't be null
     originalOnAnimationEnd?.call(this, event);
   };
+
+  dummy.addEventListener('animationcancel', () => {
```

Comment:
Why the listener from https://github.com/software-mansion/react-native-reanimated/pull/5452/files#diff-31934cfe147067532824ca4f698abb5b5582f9f8a656986f77efe30a459178f6R177-R180 isn't enough?